cmake/projects/tcl/schemes/hunter_autotools_project.cmake

```diff
@@ -0,0 +1,333 @@
+# Copyright (c) 2015 Ruslan Baratov, Alexandre Pretyman
```

Comment:
It's better to update original module instead of creating new and maintaining duplicate.

Reply:
cmake/modules/hunter_autotools_project.cmake updated